examples/system/pitop_load_configuration.py

```diff
@@ -0,0 +1,20 @@
+from time import sleep
+
+from pitop import Pitop
+
+
+# Load configuration from a previous session
+pitop = Pitop.from_file("/home/pi/my_custom_config.json")
```

Comment:
We probably want to provide a real json file, other than `alex.json`, for us to use in here? 
My idea was that in the docs you'll see the "save the configuration" example first and this one next, so it kind of makes sense in that context.

Reply:
I would ensure that we have stock JSON files for different configurations saved somewhere, so that the examples can be run and work out-of-the-box. These can simply duplicate the `dict`s that are used internally.

Reply:
Agree